README.md

```diff
@@ -12,7 +12,7 @@ Conceptlists in Concepticon:
 ## Statistics
 
 
-[![Build Status](https://travis-ci.org/lexibank/mitterhoferbena.svg?branch=master)](https://travis-ci.org/lexibank/mitterhoferbena)
+[![Build Status](https://travis-ci.org/XachaB/mitterhoferbena.svg?branch=master)](https://travis-ci.org/XachaB/mitterhoferbena)
```

Comment:
Sorry for barging in.  Remote has changed to your repository.

Reply:
Ok, I know why: I didn't have the rights to push a branch, so I thought I could do what @LinguList recommended I do in the same situation for concepticon, which is make a fork, and make a PR from my fork. Because it means the repo I compiled has my github repo as origin, this was changed automatically when compiling with cldfbench. What would be the solution here ?

Reply:
Ah, I didn't see that! Thanks for explaining. I'd say leave it as it is right now (we can simply re-generate the data later on) and I'll see whether there are any other options/solutions for this.

Reply:
Weirdly I did the same for https://github.com/lexibank/chindialectsurvey/, but it does not seem to have changed the build url. I have no idea what I did differently.

Reply:
ok, I know why again, and it might be the solution: in the case of chindialectsurvey,  I cloned the original repo (origin lexibank/chindialectsurvey), made changes, compiled, and tried to push. Push was rejected, so I forked the repo, changed to origin to mine, and pushed there. So temporarily changing the origin to the lexibank one would be a solution, albeit a hacky one.

Reply:
I think it is fine as it is. We can simply run `makecldf` immediately after the merge and all will be good. :)